core/scss/components/main-nav/_main-nav--light.scss

```diff
@@ -0,0 +1,165 @@
+@charset "UTF-8";
+
+.su-main-nav.su-main-nav--light {
+
+  .su-main-nav__toggle {
+    &[aria-expanded="true"] {
+      &::before {
+        background-color: $color-driftwood;
+      }
+
+      &:hover {
+        &::before {
+          background-color: $color-dark-red;
+        }
+      }
+    }
+  }
+
+  ul {
```

Comment:
Can we leave a code comment here about the difference between this `ul` and `> ul`. I am pretty sure I get it but it would help to be more clear as it caused me to do a doubletake while reading through.